README.md

```
@@ -51,9 +52,7 @@ The codebase is organized into the following packages:
     ```
 2.  Create a `keystore.properties` file in the root of the project and add the following properties:
     ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)